packages/ngx-deploy-npm/package.json

```diff
@@ -10,8 +10,10 @@
   "engines": {
     "node": ">=14.0.0"
   },
-  "dependencies": {
-    "@nrwl/devkit": "15.6.2"
+  "peerDependencies": {
+    "@nrwl/devkit": "^15.0.0",
+    "@nrwl/workspace": "^15.0.0",
+    "nx": "^15.0.0"
```

Comment:
This one was automatically added by nx build executor in the output `package.json`. Adding here explicitly to enforce the wide version range

Reply:
Nice catch.

I wouldn't notice about that 
![image](https://user-images.githubusercontent.com/7026066/216081824-2a295f02-9047-4a2b-829c-900e24c51ef5.png)